web-marketplace/src/components/organisms/UserAccountSettings/UserAccountSettings.ConnectField.tsx

```diff
@@ -0,0 +1,57 @@
+import { Body } from "web-components/src/components/typography";
+import ContainedButton from "web-components/src/components/buttons/ContainedButton";
+import OutlinedButton from "web-components/src/components/buttons/OutlinedButton";
+import { CopyButton } from "web-components/src/components/buttons/CopyButton";
+import React from "react";
+
+/** ConnectField is used internally by the UserAccountSettings component to
+ * display a single social or wallet connection.
+ */
+export const ConnectField = ({
+                        providerName,
+                        connect,
+                        disconnect,
+                        address,
+                      }: {
```

Comment:
some formatting issues still here
before I remember we ran linter check on commit, but it's been disabled because it was taking too much time, maybe we should add it to CI in a follow-up, wdyt?